src/streaming/controllers/StreamController.js

```diff
@@ -633,11 +635,21 @@ function StreamController() {
         });
 
         Promise.all(promises)
+            .then(() => {
+                const seekToPeriod = manifestModel.getValue().Period[seekToStream.getId()];
```

Comment:
Period is an array of periods while the `id` is the ID of the period. Accessing the info that way will not work